packages/synapse-react-client/src/components/ChallengeSubmission/AvailableEvaluationQueueList.tsx

```diff
@@ -0,0 +1,153 @@
+import { HelpOutlineTwoTone } from '@mui/icons-material'
+import {
+  Autocomplete,
+  Box,
+  List,
+  ListItem,
+  MenuItem,
+  TextField,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import { Evaluation } from '@sage-bionetworks/synapse-types'
+import { noop } from 'lodash-es'
+import React, { useState } from 'react'
+
+type TextWithHelpIconProps = {
+  text: string
+  tooltipTitle?: string
+}
+
+function TextWithHelpIcon(props: TextWithHelpIconProps) {
+  const { text, tooltipTitle } = props
+  return (
+    <>
+      {text}
+      {tooltipTitle && (
+        <Tooltip title={tooltipTitle}>
+          <HelpOutlineTwoTone
+            sx={{
+              ml: 1,
+              color: 'grey.600',
+              fontSize: '16px',
+            }}
+          />
+        </Tooltip>
+      )}
+    </>
+  )
+}
+
+type AvailableEvaluationQueueStaticListProps = Pick<
+  AvailableEvaluationQueueListProps,
+  'evaluations'
+>
+
+function AvailableEvaluationQueueStaticList(
+  props: AvailableEvaluationQueueStaticListProps,
+) {
+  const { evaluations } = props
+
+  return (
+    <Box mt={2}>
+      <Typography variant="body1">Available Evaluation Queues:</Typography>
+      <List dense={true}>
+        {evaluations.map(evaluation => {
+          return (
+            <ListItem key={evaluation.id}>
+              <TextWithHelpIcon
+                text={evaluation.name!}
+                tooltipTitle={evaluation.submissionInstructionsMessage}
+              />
+            </ListItem>
+          )
+        })}
+      </List>
```

Comment:
Design-related question -- weren't we trying to avoid showing this whole list when it's really long? Would it make sense to do a Show/Hide Available Evaluation Queues button and wrap the list in a [Collapse](https://mui.com/material-ui/transitions/#collapse)?

Reply:
Adam wanted to make sure the full list was still available to users, if they preferred to review the full list at once. I really like the Collapse idea! I'll ask Adam what he thinks